app/views/practices/show/search_terms/_category.html.erb

```diff
@@ -1,5 +1,8 @@
+<%
+  require 'securerandom'
+%>
 <a
   class="dm-tag--big--action-primary"
-  href="/search?query=<%= term %>">
+  href="/search?query=<%= term %>&key=<%= SecureRandom.uuid %>">
```

Comment:
Do we have to add a uuid here? It appears it may be affecting the search results

Reply:
ohh... search results is probably looking at the whole query string.. not just the query param - I'll look at this..  Key is to prevent duplicate entries due to turbo links.. this only happens when going to the search results from the practice show page.

Reply:
This is not in play anymore with the refactor for ahoy.